Userland/Utilities/icc.cpp

```diff
@@ -31,12 +32,24 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     Core::ArgsParser args_parser;
 
-    static StringView icc_path;
-    args_parser.add_positional_argument(icc_path, "Path to ICC profile", "FILE");
+    static StringView path;
+    args_parser.add_positional_argument(path, "Path to ICC profile or to image containing ICC profile", "FILE");
     args_parser.parse(arguments);
 
-    auto icc_file = TRY(Core::MappedFile::map(icc_path));
-    auto profile = TRY(Gfx::ICC::Profile::try_load_from_externally_owned_memory(icc_file->bytes()));
+    auto file = TRY(Core::MappedFile::map(path));
+    auto icc_bytes = file->bytes();
```

Comment:
Might be better to call this something like `file_bytes`, given we don't actually know what it is at this point, and then assign to a `ReadonlyBytes icc_bytes;` in the `if (decoder) ... else ...` branches.

Reply:
Done.